installer.sh

```diff
@@ -63,63 +63,66 @@ get_os_info() {
     esac
 }
 
-# TODO: Add option to specify which release to download.
-
 # Download a Wasp binary package and install it in $HOME_LOCAL_BIN.
 install_from_bin_package() {
-    PACKAGE_URL="https://github.com/wasp-lang/wasp/releases/latest/download/$1"
-    make_temp_dir
-    info "Downloading binary package to temporary dir and unpacking it there...\n"
-    dl_to_file "$PACKAGE_URL" "$WASP_TEMP_DIR/$1"
-    echo ""
-    mkdir -p "$WASP_TEMP_DIR/wasp"
-    if ! tar xzf "$WASP_TEMP_DIR/$1" -C "$WASP_TEMP_DIR/wasp"; then
-      die "Unpacking binary package failed."
+    BIN_PACKAGE_NAME=$1
+
+    LATEST_VERSION=`get_latest_wasp_version`
+
+    if [ -z "$VERSION_ARG" ]; then
+        VERSION_TO_INSTALL=$LATEST_VERSION
+    else
+        VERSION_TO_INSTALL=$VERSION_ARG
     fi
 
-    # TODO: Consider installing into /usr/local/bin and /usr/local/share instead,
-    #   since those are always on the PATH and are standard place to install programs like this.
-    #   But then we need to run some commands below with sudo.
-    DATA_DST_DIR="$HOME_LOCAL_SHARE"
+    if [ "$VERSION_TO_INSTALL" = "$LATEST_VERSION" ]; then
+        LATEST_VERSION_MESSAGE="latest"
+    else
+        LATEST_VERSION_MESSAGE="latest is $LATEST_VERSION"
+    fi
+
+    info "Installing wasp v$VERSION_TO_INSTALL ($LATEST_VERSION_MESSAGE).\n"
```

Comment:
Do we want to standardize on using a v prefix vs not in user-level messaging?

`Installing wasp v0.5.1.0 (latest is 0.6.0.0).`

In other places we say "version", so maybe here we say "Installing wasp version XYZ"? Super small, just noting.

Reply:
Done, now using `version`!